src/components/sidekick/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { RootState } from '../../store';
+import { connectContainer } from '../../store/redux-container';
+
+import { IfAuthenticated } from '../authentication/if-authenticated';
+import { IconButton, Icons } from '@zer0-os/zos-component-library';
```

Comment:
Is there an accompanying zOS-component-library PR for the Icons reference?

Reply:
Yes, i created the PR for that. https://github.com/zer0-os/zOS-component-library/pull/41

Reply:
I think you'll need to update the zOS-component-library version to 0.18.7 in this repositories package.json.